README.md

```diff
@@ -171,6 +171,21 @@ installed), the RPM package can be built invoking the following command:
 sh ./package-keystone-spassword.sh
```

Comment:
Do we have CNR in thi repo? Should something be added to it about the change done in this PR?